projects/graph/src/graph.py

```diff
@@ -83,6 +82,28 @@ def bft_path(self, start, target):
         ## Create a store for the path
         ## If first is in the path 
 
+    def dft_path(self, start, target):
+        print('DFT')
+        if start in self.vertices:
+            nextItems = []
+            visited = []
+            path = []
+            nextItems.append([start])
+            while len(nextItems) != 0:
+                first = nextItems.pop()
+                if first[-1] not in visited:
```

Comment:
the way that you are going about your append is a bit redundant, but in a creative use. Try to better optimize your solution, using less methods. Other than that this is great work!